docs/PowerAuth-SDK-for-Android.md

```diff
@@ -2718,6 +2719,95 @@ if (BuildConfig.DEBUG) {
+    override fun onActivationCreateSucceed(result: CreateActivationResult) {
+        if (result.userInfo != null) {
+            // User information received.
+            // At this moment, the object is also available at
+            // powerAuthSDK.lastFetchedUserInfo
+        }
+    }
+
+    override fun onActivationCreateFailed(t: Throwable) {
+        // Error handling
+    }
+})
+```
+
+To fetch the user information at a later time, use the following code:
+
+```kotlin
+val userInfo = powerAuthSDK.lastFetchedUserInfo
+if (userInfo != null) {
+    // User information is already available
+} else {
+    powerAuthSDK.fetchUserInfo(context, object : IUserInfoListener {
+        override fun onUserInfoSucceed(userInfo: UserInfo) {
+            // User information received
+        }
+
+        override fun onUserInfoFailed(t: Throwable) {
+            // Error handling
+        }
+    })
+}
+```
+
+The obtained `UserInfo` object contains the following properties:
+
+| Property                | Type     | Description |
+|-------------------------|----------|-------------|
+| `subject`               | `String` | The user's identifier |
+| `name`                  | `String` | The full name of the user |
+| `givenName`             | `String` | The given or first name of the user |
+| `familyName`            | `String` | The surname(s) or last name(s) of the user |
+| `middleName`            | `String` | The middle name of the user |
+| `nickname`              | `String` | The casual name of the user |
+| `preferredUsername`     | `String` | The username by which the user wants to be referred to at the application |
+| `profileUrl`            | `String` | The URL of the profile page for the user |
+| `pictureUrl`            | `String` | The URL of the profile picture for the user |
+| `websiteUrl`            | `String` | The URL of the user's web page or blog |
+| `email`                 | `String` | The user's preferred email address |
+| `isEmailVerified`       | `Boolean`| True if the user's email address has been verified, else false<sup>1</sup> |
+| `phoneNumber`           | `String` | The user's preferred telephone number<sup>2</sup> |
+| `isPhoneNumberVerified` | `Boolean`| True if the user's telephone number has been verified, else false<sup>1</sup> |
+| `gender`                | `String` | The user's gender |
+| `birthdate`             | `Date`   | The user's birthday |
+| `zoneInfo`              | `String` | The user's time zone, e.g. `Europe/Paris` or `America/Los_Angeles` |
+| `locale`                | `String` | The end-user's locale, represented as a BCP47 language tag<sup>3</sup> |
+| `address`               | `UserAddress` | The user's preferred postal address |
+| `updatedAt`             | `Date`   | The time the user's information was last updated |
+| `allClaims`             | `Map<String, Any>` | The full collection of standard claims received from the server |
```

Comment:
Even non-standard will be there. Generally, it should be mentioned that if the server uses custom claim (and it does in our app for example), it will be available in allClaims property.